src/modules/utils/callMetaplexIndexerRPC.ts

```diff
@@ -0,0 +1,58 @@
+import { Listing } from '@/common/components/elements/ListingPreview';
+import { toast } from 'react-toastify';
+import staticListings from 'fixtures/listings.json';
+
+const storeBlacklist: string[] = [];
+
+// will eventually be moved to server
+function initialListingFilter(listing: Listing) {
+  if (
+    listing.ended || // past listings
+    (listing.ends_at && new Date(listing.ends_at).getTime() < Date.now()) || // auctions that ended, but .ended flag not flipped
+    (!listing.ends_at && listing.last_bid) || // remove instant buys that have ended
+    (listing.ends_at && new Date(listing.ends_at).getTime() - Date.now() > 86400000 * 31) || // listings more than 31 days in the future
+    (listing?.price_floor || 0) * 0.000000001 > 2000 ||
+    storeBlacklist.includes(listing.subdomain)
+  ) {
+    return false;
+  }
+
+  return true;
+}
+
+export async function callMetaplexIndexerRPC(
```

Comment:
This is most likely a ticket for later, but we should probably look at getting a library to do JSONRPC request and response handling.

Reply:
Yeah, I do recall discussing it, I was just swamped at the time.  Down to help find a library once I'm done with the backend.